.github/scripts/trymerge.py

```diff
@@ -1204,7 +1230,7 @@ def find_matching_merge_rule(
         if not skip_internal_checks and pr.has_internal_changes():
             raise RuntimeError("This PR has internal changes and must be landed via Phabricator")
 
-        return rule
+        return (rule, pending_checks, failed_checks)
```

Comment:
I think find_matching_merge_rule would only save the checks for the rule, not all the checks, so I think stuff like trunk might get lost.  Also, the lists might not be complete if force is used

It might be better to use pr.get_conclusions or something and parse it out separately

Reply:
Yeah, this is the exact problem that I have just discovered.  I need to actually run the test on this PR to notice that.  Let me add `ciflow/trunk` and test it again.  The pending trunk jobs should show up on Rockset.